README.md

```diff
@@ -1 +1,127 @@
-# Leaderboard-list
\ No newline at end of file
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [Future Features](#future-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Test](#test)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Leaderboard Application<a name="about-project"></a>
+
+**Leaderboard Application** The leaderboard website displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to the external [Leaderboard API service](https://www.notion.so/microverse/Leaderboard-API-service-24c0c3c116974ac49488d4eb0267ade3).
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/">HTML</a></li>
+    <li><a href="https://www.w3schools.com/css/">CSS</a></li>
+    <li><a href="https://www.w3schools.com/js/">JavaScript</a></li>
+    <li><a href="https://www.w3schools.com/WebPack/">Webpack</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Add new names with their score.
+- Display a list of people.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Future Features <a name="future-features"></a>
+
+- **Implement a CSS Preprocessor.**
+- **Implement a Database**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Setup
+
+To get a local copy up and running, follow these steps:
+
+1. Clone the repository: `git clone git@github.com:ulises2607/Leaderboard-list.git`
+2. Navigate to the project directory: `cd Leaderboard-list`
+3. Install the dependencies: `npm install`
+4. Build the project: `npm run build`
+5. Open `index.html` in your web browser.
+
+Then, you will change everything you want!
+
+### Test
+
+In order to follow the Linters you could check them using this:
+
+```
+HTML Checker:
+npx hint .
+
+CSS Checker:
+npx stylelint "**/*.{css,scss}"
+
+JavaScript Checker:
+npx eslint .
+```
+
+#### Note
+
+Use at the end '--fix' to help you out with any error.
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Rysth**
+
+- GitHub: [@ulises2607](https://github.com/ulises2607)
+- LinkedIn: [Cesar Ulises Cappa Subelza](https://www.linkedin.com/in/cesar-ulises-cappa-subelza/)
+
```

Comment:
![image](https://github.com/ulises2607/Leaderboard-list/assets/93039359/b43e9c68-7c10-4bcb-83e4-a97b084aecdf)

- [ ] It would be great if you fix your LinkedIn link because right now it is broken 😨 and showing an error which you can see in the image above ⬆️ Kindly fix this link to make your readme descriptive and professional 👍 😊